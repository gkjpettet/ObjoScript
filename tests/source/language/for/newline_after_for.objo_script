for // expect error as condition on new line.
(var i = 0; i < 10; i++) {
System.print(i)
}