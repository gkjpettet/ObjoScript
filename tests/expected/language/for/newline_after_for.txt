Expected a `(` after the `for` keyword.